components/faqs/reverse-selectors.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { withTheme, ThemeProvider } from 'styled-components'
+
+import SectionLayout from '../SectionLayout'
+import LiveEdit from '../LiveEdit'
+
+const scope = { withTheme, ThemeProvider }
+const sample = (`
+const Link = styled.a\`
+  display:     flex;
+  align-items: center;
+  padding:     5px 10px;
+  background:  papayawhip;
+  color:       palevioletred;
+\`;
+
+const Icon = styled.svg\`
+  transition: fill 0.25s;
+  width:      48px;
+  height:     48px;
+
+  \${Link}:hover & {
+    fill: rebeccapurple;
+  }
+\`;
+
+const Label = styled.span\`
+  display:     flex;
+  align-items: center;
+
+  line-height: 1.2;
+  
+  &::before {
+    content: '◀';
+    margin: 0 10px;
+  }
+\`;
+
+render(
+  <Link href="#">
+    <Icon viewBox="0 0 512 512">
+      <path d="M233.288,197.742l-49.948-48.33c-3.136-3.034-7.999-3.377-11.53-0.811l-10.189,7.409l-36.874-50.718l27.151-19.74 c2.032-1.477,3.353-3.737,3.642-6.233c0.289-2.495-0.479-4.997-2.12-6.9L100.1,10.565C94.312,3.851,85.904,0,77.032,0    c-6.466,0-12.653,2.012-17.89,5.82L35.818,22.779c-12.708,9.239-16.272,26.995-8.116,40.425l42.397,69.793    c1.305,2.147,3.448,3.649,5.91,4.143c0.587,0.119,1.179,0.175,1.769,0.175c1.883,0,3.736-0.592,5.283-1.718l27.15-19.739    l36.875,50.717l-10.189,7.409c-3.53,2.567-4.705,7.299-2.785,11.218l30.571,62.418c1.168,2.384,3.328,4.129,5.904,4.768    c0.715,0.177,1.441,0.265,2.165,0.265c1.88,0,3.733-0.591,5.284-1.718l54.289-39.47c2.147-1.561,3.493-3.99,3.679-6.637    C236.19,202.18,235.195,199.587,233.288,197.742z M106.918,96.038c-0.002,0.001-0.005,0.002-0.007,0.005    c-0.002,0.001-0.004,0.004-0.006,0.005l-26.516,19.279l-37.33-61.451c-3.342-5.502-1.882-12.778,3.325-16.563l23.324-16.958 c2.145-1.561,4.679-2.385,7.324-2.385c3.638,0,7.086,1.578,9.458,4.329l46.945,54.46L106.918,96.038z M176.116,230.114    l-22.548-46.037l22.701-16.503l36.838,35.644L176.116,230.114z"/>
```

Comment:
This is extremely pretty, but the code is waaaaaayyyyy too long :) Remember that this actually shows up and is editable.